frontends/web/src/routes/settings/electrum-servers.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ElectrumServer } from './electrum-server';
+import { getConfig, getDefaultConfig, setConfig } from '../../api/backend';
+import { confirmation } from '../../components/confirm/Confirm';
+import style from './electrum.module.css';
+import A from '../../components/anchor/anchor';
+
+type Props = {
+  coin: 'btc' | 'tbtc' | 'ltc' | 'tltc';
+};
+
+interface IElectrumServer {
+  server: string;
+  tls: boolean;
+  pemCert: string;
+}
+
+export const ElectrumServers = ({
+  coin
+}: Props) => {
+  const { t } = useTranslation();
+  const [config, setConfigState] = useState<any>();
+  const loadConfig = () => {
+    getConfig().then(setConfigState);
+  };
+  useEffect(loadConfig, []);
+  if (config === undefined) {
+    return null;
+  }
+  const electrumServers: IElectrumServer[] = config.backend[coin].electrumServers;
+
+  const save = async (newElectrumServers: IElectrumServer[]) => {
+    const currentConfig = await getConfig();
+    currentConfig.backend[coin].electrumServers = newElectrumServers;
+    await setConfig(currentConfig);
+    setConfigState(currentConfig);
+  };
+
+  const onAdd = (server: IElectrumServer) => {
+    let newElectrumServers = electrumServers.slice();
+    newElectrumServers.push(server);
```

Comment:
Unsure if this is a part of the PR, but might as well refactor this while we're at it.

```suggestion
    const newElectrumServers = [...electrumServers, server];
```

Reply:
Neat! Done